src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/CaseCloseValidator.java

```diff
@@ -24,4 +27,18 @@
         return errors;
     }
 
+    public static List<String> validateBfActionsForCaseCloseEvent(CaseData caseData) {
+        List<String> errors = new ArrayList<>();
+        if (CollectionUtils.isEmpty(caseData.getBfActions())) {
+            return errors;
+        }
+        for (var currentBFActionTypeItem : caseData.getBfActions()) {
+            if (currentBFActionTypeItem.getValue().getCleared() == null) {
+                errors.add(CLOSING_CASE_WITH_BF_OPEN_ERROR);
+                return errors;
```

Comment:
if a case had 2 bfs which were both uncleared, this would return only after the first. This return can be removed